Sources/Views/ALKInformationCell.swift

```diff
@@ -76,31 +84,110 @@ final class ALKInformationCell: UITableViewCell {
+    }
+    
+    fileprivate func setUpConstraintsForRating() {
+        
+        let horizontalStackView = UIStackView()
+        let verticalStackView = UIStackView()
+        let lineViewLeft = UIView()
+        let lineViewRight = UIView()
+        
+        verticalStackView.axis  = NSLayoutConstraint.Axis.vertical
+        verticalStackView.distribution  = UIStackView.Distribution.equalSpacing
+        verticalStackView.alignment = UIStackView.Alignment.center
+        verticalStackView.spacing   = -5
+        
+        verticalStackView.addArrangedSubview(messageView)
+        if !commentTextView.text.isEmpty {
+            verticalStackView.addArrangedSubview(commentTextView)
+        }else{
+            verticalStackView.removeArrangedSubview(commentTextView)
+        }
+        
+        horizontalStackView.axis  = NSLayoutConstraint.Axis.horizontal
+        horizontalStackView.distribution  = UIStackView.Distribution.equalSpacing
+        horizontalStackView.alignment = UIStackView.Alignment.center
+        horizontalStackView.spacing   = 10
+
+        horizontalStackView.addArrangedSubview(lineViewLeft)
+        horizontalStackView.addArrangedSubview(verticalStackView)
+        horizontalStackView.addArrangedSubview(lineViewRight)
+        
+        lineViewLeft.backgroundColor = .lightGray
+        lineViewRight.backgroundColor = .lightGray
+        
+        contentView.addViewsForAutolayout(views: [horizontalStackView])
+        contentView.bringSubviewToFront(messageView)
+        
+        horizontalStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 10).isActive = true
+        horizontalStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10).isActive = true
+        
+        lineViewLeft.widthAnchor.constraint(equalToConstant: 60).isActive = true
+        lineViewRight.widthAnchor.constraint(equalToConstant: 60).isActive = true
+        lineViewLeft.heightAnchor.constraint(equalToConstant: 1).isActive = true
+        lineViewRight.heightAnchor.constraint(equalToConstant: 1).isActive = true
     }
 
     func setupStyle() {
         contentView.backgroundColor = UIColor.clear
         backgroundColor = UIColor.clear
-
-        bubbleView.backgroundColor = ALKMessageStyle.infoMessage.background
+        
         messageView.setFont(ALKMessageStyle.infoMessage.font)
         messageView.textColor = ALKMessageStyle.infoMessage.text
+        commentTextView.setFont(ALKMessageStyle.feedbackComment.font)
+        commentTextView.textColor = ALKMessageStyle.feedbackComment.text
+    }
+    
+    func getFeedback(viewModel: ALKMessageViewModel) -> Dictionary<String,Any>? {
+        guard let feedbackString = viewModel.metadata?["feedback"] as? String else { return nil }
+        guard let dictionary = try? JSONSerialization.jsonObject(with: feedbackString.data(using: .utf8)!, options: []) as? [String:Any] else { return nil }
```

Comment:
Don't force unwrap the feedback string data.

Reply:
done